finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,93 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "J-Dawg";
+var specialAbility = "transports herself to another location in entire world with a sneeze";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "What's up! I'm J-Dawg!";
+var catchPhrase = "Hello, Achoo!, Goodbye!";
 
 // Declare two variables - power AND energy - set to integers
+var power = 1313;
+var energy = 7850;
 
 // Declare two variables - fullPower AND fullEnergy
+var fullPower = (power * 500 );
+var fullEnergy = (energy + 150);
+
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
+var archEnemies = ["Glueman", "The Timestopper", "Sheilder"] ;
+var sidekicks = ["The Flash", "Superman", "Superwoman"];
 
 // Print the first sidekick to your console
+console.log(sidekicks[0]);
+
 
 // Print the last archEnemy to the console
+console.log(archEnemies[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+var count = archEnemies.push("Dr. Passport Enforcer");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+console.log(sidekicks.pop());
```

Comment:
2 notes on this 
- Double check what `pop` does!
- When you call a method on an array, we usually want to do that outside of a `console.log`. We can always log the updated array on the next line. Example:

```javascript
sidekicks.pop();
console.log(sidekicks);
```